### PR TITLE
New Repo Restructuring

### PR DESCRIPTION
**Pull request by jiaccarino**
Update Phase 1 Directory Structure
2 new metrics:
- Dependency Pinning
- Code Review
image
image

Added Backed:
image

Added Backend Tests:
image

Added Frontend:
http://ec2-3-87-4-91.compute-1.amazonaws.com:3000/

Added CI/CD

### CODE REVIEW

File: package.txt
Comment (glenscheng) [inline package.txt:1]: what is this from?

File: phase1/dist/api/apiUtils.d.ts
Comment (glenscheng) [inline phase1/dist/api/apiUtils.d.ts:1]: Should we delete the dist/ directory again?

File: phase1/test/index.test.d.ts
Comment (glenscheng) [inline phase1/test/index.test.d.ts:1]: there also seems to be dist files in phase1/test/ directory

**Review by glenscheng — COMMENTED** [review]
made comments

File: package.txt
Comment (jiaccarino) [reply package.txt:1]: it was from updating the new package.json & stashing the old one
will get rid of it

File: phase1/dist/api/apiUtils.d.ts
Comment (jiaccarino) [reply phase1/dist/api/apiUtils.d.ts:1]: yes just did

File: phase1/test/index.test.d.ts
Comment (jiaccarino) [reply phase1/test/index.test.d.ts:1]: deleted these as well

**Review by glenscheng — APPROVED** [review]
LGTM! all my comments were addressed

**Review by matthewl121 — APPROVED** [review]
PR looks good to merge to main

**Pull request merged**